monai/networks/nets/dynunet.py

```diff
@@ -32,7 +33,7 @@ class DynUNetSkipLayer(nn.Module):
     forward passes of the network.
     """
 
-    heads: Optional[List[torch.Tensor]]
+    heads: Optional[list[torch.Tensor]]
```

Comment:
A feasible way may be revert the change in this file or try to set `Union[None, List[Tensor]]` due to TorchScript has limited support for type annotations, especially when dealing with complex types, such as nested generic types.